tools/prepare_mapping_v1.py

```
@@ -1,136 +1,179 @@
-ws2.append(["", "semantic"])
-ws2.append(["", "functional"])
-ws2.append(["strength_of_relationship", "use an integer or empty value"])
-
-ws3 = wb_output.create_sheet("source")
-ws3.append(["node_id", "assessable", "urn", "ref_id", "name", "description"])
-for node in source["objects"]["framework"]["requirement_nodes"]:
-    node_id = node["urn"].split(":")[-1] if node["assessable"] else ""
-    ws3.append(
+
+
+def generate_mapping_excel(source_yaml, target_yaml, compat=False):
```

Pylint (3.3.7)

[refactor] Too many local variables (31/15)

(R0914)